"Translated" icon.gif to svg

ralferlebach opened the pull request:
Since Moodle 2.6, icons are delivered to the system in svg format in order to be scalable freely. I rebuild your icon into svg, using the moodle color scheme.

I hope, this helps. :-)

Commit:
"Translated" icon.gif to svg

Since Moodle 2.6, icons are delivered to the system in svg format in order to be scalable freely. I rebuild your icon into svg, using the moodle color scheme.

I hope, this helps. :-)

Aequalitas commented:
Hey Ralf,
thanks for this contribution, i wasnt aware of the possibility of svg.

I tried it out myself and it works but at the plugin overview as an admin, it doesnt show up there.
Do you get the same issue?

Thanks,
Aequalitas

ralferlebach commented:
Hi,

after changing the version number in version.php (e.g. from 2018120400 to 2018120401) Moodle (from 2.4 on) will automatically change use to svg. I put a screenshot of my live system here.
![screenshot courseboard in course view](https://cloud.githubusercontent.com/assets/12737344/8220778/2e2d12c6-1557-11e5-8b6a-7460386a21d0.png)

By the way, the version number itsself looks a little bit odd, as it should read like 2015mmddxx (year, month, day, subversion) - I believe, changing the version number back from 2018###### to 2015###### won't let moodle upgrade or update the mod - so it would be advicable to stay with 2018 until the year 2018 is over. ;-)

Best regards

Ralf

PS: I hope I understood your question right and my answer fits. I add also a screen shot of the plugin overview page (as there also everything seems to be fine...)
![screenshot courseboard in plugin overview](https://cloud.githubusercontent.com/assets/12737344/8220782/3b89cc16-1557-11e5-9325-ba54ed82481b.png)

PS 2: I also managed to get the courseboard module runnig on Moodle 2.6.11 flawlessly just by setting the required moodle version in version.php to the current version of this branch, (In that case 2012062500)

Commit:
Update mod_form.php

Made it compatible to Moodle 2.9 (and upward)

(moodleform_mod::add_intro_editor() is considered deprecated)

Commit:
Update mod_form.php

Aequalitas commented:
Im using the newest version(2.9) and it doesnt show up in the plugin overview.
Here a screenshot so you see what i mean.

![svgiconmissing](https://cloud.githubusercontent.com/assets/7952093/8316033/b3613282-19f4-11e5-9519-b4e56c803cc8.jpg)
